subsys/nrf_security/src/core/lite/psa_core_lite.c

```diff
@@ -142,6 +142,49 @@ psa_status_t psa_verify_message(
 
 #if defined(PSA_WANT_ALG_SHA_256) || defined(PSA_WANT_ALG_SHA_512)
 
+psa_status_t psa_hash_setup(psa_hash_operation_t *operation, psa_algorithm_t alg)
+{
+	if (operation == NULL) {
+		return PSA_ERROR_INVALID_ARGUMENT;
+	}
+
+	if (!UTIL_CONCAT_OR(VERIFY_ALG_SHA_256(alg),
+			    VERIFY_ALG_SHA_512(alg))) {
+		return PSA_ERROR_NOT_SUPPORTED;
+	}
+
+	return psa_driver_wrapper_hash_setup(operation, alg);
+}
+
+psa_status_t psa_hash_update(
+	psa_hash_operation_t *operation, const uint8_t *input, size_t input_length)
+{
+	if (operation == NULL || input == NULL) {
+		return PSA_ERROR_INVALID_ARGUMENT;
+	}
+
+	return psa_driver_wrapper_hash_update(operation, input, input_length);
+}
```

Comment:
I believe input  `NULL` is normally valid? At least that is how we treat it in CRACEN

Reply:
Makes sense